components/SignupForm/SignupForm.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { Layout } from "../Layout";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import styled from "@emotion/styled";
+import {
+  AuthFormType,
+  loginForm,
```

Comment:
We're importing loginForm, forgotPasswordForm, updatePasswordForm and updateEmailForm but not using these - let's remove these lines